fake_dependencies/capability-service/server.js

```diff
@@ -42,6 +43,9 @@ app.post("/api/v1/capabilities", (req, res) => {
         newTeam.description = "generic description"
     }
 
+    // RootId generation
+    newTeam.RootId = (newTeam.name.substring(0, 20) + "-" + uuidv1().substring(0, 8)).toLocaleLowerCase();

```

Comment:
technically speaking, this generates incompatible rootIds but it shouldn't matter for this particular purpose. If we get around to it, we can use this instead https://hashids.org/javascript/